yarn-project/noir-contracts/contracts/inclusion_proofs_contract/src/main.nr

```diff
@@ -140,15 +147,19 @@ contract InclusionProofs {
 
         // 3) Compute the nullifier from the note
         if maybe_note.is_some() {
+            // docs:start:prove_note_not_nullified
             prove_note_not_nullified(
                 ValueNoteMethods,
```

Comment:
are ok to assume that user would know what `ValueNoteMethods` are? Since in your example you do not import it or explain what these are

Reply:
I think if the user has got to this point we can assume they know what `ValueNoteMethods` are.  as it's a how-to and not a tutorial we try to explain bare minimum. but would be good to get some input from @critesjosh

Reply:
fair -> what I meant was add it to the imports section

Reply:
good idea, done